lib/puppet/provider/firewall/iptables.rb

```diff
@@ -375,6 +375,11 @@ def self.instances
   end
 
   def self.rule_to_hash(line, table, counter)
+    if line =~ /^-A KUBE-(SERVICES|SVC|SEP)/
```

Comment:
TBH, we should probably just be broader than this and not bother filtering SERVICES/SVC/SEP. They've added new chains over time and we can avoid that becoming a mysterious regression in the future. To avoid conflict with kube-proxy we already should to avoid KUBE- chain names for our own stuff